test/levels.test.js

```diff
@@ -0,0 +1,92 @@
+import _ from 'lodash';
+import levels from '../src/levels';
+
+describe('levels', () => {
+  it('should be defined', () => {
+    let expectedLevels = [
```

Comment:
in what way are these "expected" ? I define them in another file, and now we have a test that asserts that what I wrote will not change unless I change this file as well? 🤔